src/core/server/server.api.md

```diff
@@ -515,6 +532,14 @@ export class SavedObjectsClient {
 // @public
 export type SavedObjectsClientContract = Pick<SavedObjectsClient, keyof SavedObjectsClient>;
 
+// Warning: (ae-missing-release-tag) "SavedObjectsClientProviderOptions" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)

```

Comment:
Please add a doc comment block above every exported type like:
```
/**
 * @public
 */
```

Bonus points for adding a one liner to explain what the type does 😄

Reply:
You got it!